identity_credential/src/credential/jws.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020-2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+/// A wrapper around a JSON Web Signature (JWS).
+pub struct Jws(String);
```

Comment:
```suggestion
#[derive(Debug, Clone)
pub struct Jws(String);
```

Add the usual suspects.